desc/objectives/normalization.py

```diff
@@ -41,11 +41,7 @@ def get_lowest_mode(basis, coeffs):
         scales["A"] = np.pi * scales["a"] ** 2
         scales["V"] = 2 * np.pi * scales["R0"] * scales["A"]
         scales["B_T"] = abs(thing.Psi) / scales["A"]
-        iota_avg = np.mean(np.abs(thing.get_profile("iota")(np.linspace(0, 1, 20))))
-        if np.isclose(iota_avg, 0):
-            scales["B_P"] = scales["B_T"]
-        else:
-            scales["B_P"] = scales["B_T"] * iota_avg
+        scales["B_P"] = scales["B_T"]
```

Comment:
I think we should make it even simpler, idt anyone will ever use B_T and B_P as the scales for a problem, so we can just do `scales["B"] = abs(thing.Psi) / scales["A"]` and get rid of the B_T and B_P keys entirely